tests/server/test_server_status.py

```diff
@@ -44,26 +42,7 @@ async def test_server_status(server, client):
 
 @pytest.mark.asyncio
 async def test_server_status_database_unreachable(server, client):
-    await Environment.close_connection_pool()
-    result = await client.get_server_status()
-    assert result.code == 200
-    database_slice = None
-    for slice in result.result["data"]["slices"]:
-        if slice["name"] == "core.database":
-            database_slice = slice
-    assert database_slice
-    assert not database_slice["status"]["connected"]
-
-
-@pytest.mark.asyncio
-async def test_server_status_database_down(
```

Comment:
This test case does not work with a remove database. I did not make it conditional because the value is not very high